collect_app/src/main/res/layout/storage_migration_dialog.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="?attr/colorSurface"
+    android:orientation="vertical">
+
+    <include layout="@layout/material_full_screen_dialog_toolbar" />
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical"
+            android:padding="@dimen/margin_standard">
+
+            <TextView
+                android:id="@+id/messageText1"
+                style="@style/TextAppearance.Collect.Body1"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/margin_small"
+                android:text="@string/storage_migration_dialog_message1" />
+
+            <TextView
+                android:id="@+id/messageText2"
+                style="@style/TextAppearance.Collect.Body1"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:visibility="gone"
+                android:layout_marginTop="@dimen/margin_small"
+                android:text="@string/storage_migration_dialog_message2" />
+
+            <TextView
+                android:id="@+id/messageText3"
+                style="@style/TextAppearance.Collect.Body1"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/margin_small"
+                android:text="@string/storage_migration_dialog_message3" />
+
+            <TextView
```

Comment:
This can be a `MaterialButton` with the `TextButton` style right?

Reply:
Yes but such a text button always has it's own padding and it doesn't look good because it's not aligned with other elements (`TextViews`).

Reply:
Ah because of the left/right padding? Personally I think I'd rather we use the component and style and then customize the padding or margins around it to get it to look right. That way most of the styling still comes from the Material Components lib.

Reply:
Done.